go/private/test.bzl

```diff
@@ -66,7 +75,7 @@ def _go_test_impl(ctx):
       library = None,
       want_coverage = False,
       importpath = ctx.label.name + "~testmain~",
-      golibs = [golib],
+      golibs = [golib] + covered_libs,
```

Comment:
We'll end up passing `golib` in multiple times here. This could go in another CL, but I think `emit_library_actions` should accept a depset instead of a list for this parameter.